src/main/java/com/dreammaster/gthandler/recipes/SpaceAssemblerRecipes.java

```diff
@@ -12,23 +12,22 @@
 import static gregtech.api.util.GT_RecipeBuilder.MINUTES;
 import static gregtech.api.util.GT_RecipeBuilder.SECONDS;
 
-import net.minecraft.item.ItemStack;
-import net.minecraft.nbt.NBTTagCompound;
-import net.minecraftforge.fluids.Fluid;
-import net.minecraftforge.fluids.FluidRegistry;
-import net.minecraftforge.fluids.FluidStack;
-
+import com.dreammaster.item.ItemList;
 import com.github.technus.tectech.thing.CustomItemList;
 import com.gtnewhorizons.gtnhintergalactic.recipe.IG_RecipeAdder;
 
-import gregtech.api.enums.ItemList;
```

Comment:
You removed the ItemList from GT5u and included the ItemList from CoreMod - yes these should have separate names but this means you cannot access the item enums from GT5u.

You can circumvent this by using the full name of either list (I suggest CoreMod, as the recipe code already does this, which makes me think your IDE pulled a prank on you.)

Example:
`com.dreammaster.item.ItemList.CircuitUXV.getIS(4)`